src/dndbeyond/content-scripts/character.js

```diff
@@ -432,9 +453,16 @@ function rollItem(force_display = false, force_to_hit_only = false, force_damage
             damages.push(cleric_level < 14 ? "1d8" : "2d8");
             damage_types.push("Divine Strike");
         }
+        // Cleric Blessed strikes
+        if (character.hasClassFeature("Blessed Strikes") &&
+            character.getSetting("cleric-blessed-strikes", false)) {
+            damages.push("1d8");
+            damage_types.push("Blessed Strikes");
+        }
         // Bard's Psychic blades;
         if (character.hasClassFeature("Psychic Blades") &&
-            character.getSetting("bard-psychic-blades", false)) {
+            character.getSetting("bard-psychic-blades", false) &&
+            character.hasClass("Bard")) {
```

Comment:
Can you have a class feature of Psychic Blades without being a Bard?  Is that some new feature in Tasha's ?

Reply:
Yeah, thank WotC for this crap:
Rogue Psychic Blades: https://www.dndbeyond.com/sources/tcoe/rogue#PsychicBlades
Bard College of Whispers Psychic Blades: https://www.dndbeyond.com/classes/bard#CollegeofWhispers

Just, ya know, named the same dang stupid exact same name...

Reply:
oh cool, so.. who cares about multiclassers!

Reply:
They'll still be good, as it catches only for Bard here, and they'd have that if they multi-classed bards & rogues.

Unless you're mouthing off at WotC, if so I AGREE!

Reply:
yeah, I was mouthing off 😁 
But if someone has both rogue psychic blades and bard psychic blades then they're SOL.

Reply:
Shouldn't be, the part for the Rogue is elsewhere in the code

Reply:
eh, right, I'm stupid, I was thinking of rolling "Psychic Blades" not of rolling something while also having the feat.
How about you're a Bard, but not yet 3rd level, or not of the right college, and you have rogue psychic blades.
(note: I'm just thinking of issues that I have no intentions of fixing. There's an option for that feat anyway, just annoyed at WotC crap)

Reply:
yeah, that could be an issue... they'd have to have it toggled on...

I say screw it as well, the DM and Player should know what the things be